### DISCUSSION TITLE
不用找群啦，来这里分享下自己的首页吧

### DISCUSSION THREAD

**Discussion by avsba001** [hslr-s/sun-panel#69] [General]
图来自b站
![image](https://github.com/hslr-s/sun-panel/assets/29015932/93814d5c-f3be-46f0-b0c5-8f505fa62db6)

**Comment by zero617** [comment]
> 图来自b站 ![image](https://private-user-images.githubusercontent.com/29015932/298272021-93814d5c-f3be-46f0-b0c5-8f505fa62db6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU5Mzc0NzAsIm5iZiI6MTcwNTkzNzE3MCwicGF0aCI6Ii8yOTAxNTkzMi8yOTgyNzIwMjEtOTM4MTRkNWMtZjNiZS00NmYwLWIwYzUtOGY1MDVmYTYyZGI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIyVDE1MjYxMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWJiNmMwY2Y4NzdhN2U1ZGE4NGM1OTgxZWI4MmEyMmQzYzY4ZTdjOTFmYWFiNjNjYTBjMTYwM2I2MTc1ZGY1YTkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.LTdpa1RDZxygVLUnmeU31D26hC5cbYnciKsl0atwJo4)

这个动态背景是怎么实现的？

**Reply by ym1123** [reply]
基于gif实现的

**Comment by ym1123** [comment]
http://nav.yumo.icu/
这是我的导航页

**Comment by wangscsmile** [comment]
https://ss.wscsmile.cn:666     自己用的，端口号懒得隐藏了

**Comment by sfeii** [comment]
![捕获](https://github.com/user-attachments/assets/21bb6cec-7af6-43ea-a390-6e88ae564031)
我的资源导航站https://dd.sfeii.cn
